crates/nu-parser/src/lex.rs

```diff
@@ -395,9 +397,13 @@ fn lex_raw_string(
         *curr_offset += 1
     }
     if !matches {
+        let mut expected = '\''.to_string();
+        for _ in 0..prefix_sharp_cnt {
+            expected.push('#');
+        }
```

Comment:
```suggestion
        expected.push_str(&"#".repeat(prefix_sharp_cnt));
```

Something like this might be simpler.

Reply:
Thanks!  I fell into trouble to make it simpler :)